packages/strapi-plugin-content-manager/admin/src/containers/EditView/Header.js

```diff
@@ -28,23 +28,28 @@ const Header = () => {
     resetData,
     setIsSubmitting,
     slug,
+    clearData,
   } = useDataManager();
+  const isSingleType = useRouteMatch('/plugins/content-manager/singleType');
```

Comment:
Well here you get either `null` or an `object` so in order to make sure you retrieve the `contentType` param from the pathname you can do the following:

```
const {
    params: { contentType },
  } = useRouteMatch('/plugins/content-manager/:contentType');
const isSingleType = contentType === 'singleType';
```

Reply:
You can also adapt your condition to make sure `isSingleType` is a `Boolean`:

```
const isSingleType = useRouteMatch('/plugins/content-manager/singleType') !== null;
```

Reply:
Isn't the same as `null = false` and `{} = true` ?

Reply:
Anyway, I will make the modification it is more maintainable 👍

Reply:
Well it's more about a code convention... from a reader point of view you expect to have a `boolean` when you constant starts with `['is', 'has', 'does', ...]`.